engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/ModelData.scala

```diff
@@ -62,6 +62,10 @@ trait ModelData extends ConfigCreatorSignalDispatcher {
 
   lazy val validator: ProcessValidator = ProcessValidator.default(processWithObjectsDefinition, modelClassLoader.classLoader)
 
+  lazy val processDefinitionCategories: List[String] = {
```

Comment:
maybe let's remove it now as it's not used?

Reply:
Removed